### PR TITLE
bytesize transcript coproID

### PR DESCRIPTION
**Pull request by FranBonath**

### CODE REVIEW

File: markdown/events/2021/bytesize-29-nf-core-coproid.md [hunk 1]
```diff
@@ -25,3 +25,48 @@ It is our hope that these talks / videos will build an archive of training mater
...
+
+[11:32](https://youtu.be/gU4jx1pb8Tw&t=692)
```
Comment (atrigila) [inline markdown/events/2021/bytesize-29-nf-core-coproid.md:59]:
```suggestion
[11:39](https://youtu.be/gU4jx1pb8Tw&t=692)
```

**Review by atrigila — CHANGES_REQUESTED** [review]
For Hackaton2023, I reviewed the content of this transcript. The text is correct. I would only suggest one minor correction in a timestamp :)

**Review by mribeirodantas — APPROVED** [review]

**Comment by mribeirodantas** [comment]
Great, @atrigila ! Thanks for the contribution 🥳

**Review by gleisonm — CHANGES_REQUESTED** [review]

**Comment by FranBonath** [comment]
Hej @gleisonm Thank you very much for your contribution. I know that the changes you requested are more true to the recording than the original transcript, however, these changes were made deliberately to improve readability. Do you think they do change the original meaning of the text?

**Comment by gleisonm** [comment]
Hi, Franziska, of course no, improve the readability is more important!
Actually i had the same doubt, about that, be true to the recording, or be
more understandable, then a did the changes to know what you think too.
After all, thanks for reaching me 😊.


Em ter., 28 de mar. de 2023 às 12:29, Franziska Bonath <
***@***.***> escreveu:

**Review by atrigila — APPROVED** [review]

**Pull request merged**